themes/default/content/blog/nodejs-binaries-with-pkg/index.md

```diff
@@ -0,0 +1,151 @@
+---
+title: "Node.js Native Binary Compilation Using vercel/pkg"
```

Comment:
I noticed that you use both `vercel/pkg` and `Vercel/pkg` in this post. Which is correct?